test/services/SicCodeService.test.ts

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import { SicCodeService } from "../../src/services/SicCodeService";
+import { generateTestData } from '../TestData';
+import { StatusCodes } from 'http-status-codes';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+mockedAxios.create = jest.fn(() => mockedAxios);
+
+describe('SicCodeService', () => {
+
+    describe("when API call is successful", () => {
+        it("should return search results list", async () => {
+        // given
+        const testData = generateTestData();
+        const searchString = 'ABC';
+        const matchPhrase = false;
+
+        mockedAxios.post.mockImplementationOnce(() => Promise.resolve({
+            status: StatusCodes.OK,
+            data: testData
+          }));
+
+        // when
+        const sicCodeService = new SicCodeService();
```

Comment:
Could the instantiation of a `SicCodeService` be provided in a `beforeEach` function to avoid duplicating the instantiation for each test?